app/models/restaurant.rb

```diff
@@ -13,7 +10,7 @@ class Restaurant < ApplicationRecord
 
   validates :rating, numericality: {greater_than_or_equal_to: 0, less_than_or_equal_to: 3}
 
-  after_initialize :initialize_defaults
+  after_initialize :update_defaults
```

Comment:
Note that this is called also when you create an instance of the model by querying it from the DB.

So 'find' also calls this, for example.

I don't think it causes any harm in your case, but if you wished to limit this to creation only, you could have added `, if: :new_record?`